src/store/index.js

```diff
@@ -2,16 +2,43 @@ import Vue from 'vue'
 import Vuex from 'vuex'
 import modules from './modules'
 import EpochChain from '@aeternity/aepp-sdk/es/chain/epoch'
+import createPersistedState from 'vuex-persistedstate'
 import { wrapActionsWithResolvedEpoch } from './utils'
 
 Vue.use(Vuex)
-
 const store = new Vuex.Store({
+  plugins: [
+    createPersistedState({
+      key: 'localNetwork',
```

Comment:
I don't think that this key should be changed, maybe in future, we will store in local storage something more than a list of added networks.